ios/Capacitor/Capacitor/CAPLog.swift

```diff
@@ -1,26 +1,19 @@
 public class CAPLog {
-    public static var enableLogging: Bool = true
-    
-    private static var oneTimeConfigCheck: () -> () = {
-        // `dispatch_once` is not available since Swift 3. but, since static properties are implicitly lazy,
-        // this code will only execute once which is sufficient for our needs here. but since we need to do an
-        // async dispatch, there is a window of time where the default value will be valid before the config
-        // value(s) can be loaded.
-        DispatchQueue.main.async {
-            let config = CAPConfig()
-            if let configFlag = (config.getValue("ios.hideLogs") as? Bool) ?? (config.getValue("hideLogs") as? Bool) {
-                enableLogging = !configFlag
-            }
-        }
-        return {}
-    }()
-    
+
+    public static let config = CAPConfig()
+
     public static func print(_ items: Any..., separator: String = " ", terminator: String = "\n") {
-        oneTimeConfigCheck()
-        if enableLogging {
+        if !self.hideLogs() {
```

Comment:
I really don't understand why this is checked dynamically on every single log call — it's enormously wasteful, because the value can't change during runtime. It really should be a static value.

Reply:
This is reverting a workaround for a bug that appeared during testing (a different workaround is used for the time being). Eventually, it will be a static flag as configuration is revisited in support of https://github.com/ionic-team/capacitor/issues/3182

Reply:
Cool, thanks for letting me know.